rhcos.sh

```diff
@@ -6,6 +6,15 @@ OPENSHIFT_INSTALLER_MACHINE_OS=${OPENSHIFT_INSTALLER_MACHINE_OS:-https://raw.git
 
 # Get the rhcos.json for that commit, and find the baseURI and openstack image path
 MACHINE_OS_IMAGE_JSON=$(curl "${OPENSHIFT_INSTALLER_MACHINE_OS}")
+
 export MACHINE_OS_INSTALLER_IMAGE_URL=$(echo "${MACHINE_OS_IMAGE_JSON}" | jq -r '.baseURI + .images.openstack.path')
+export MACHINE_OS_INSTALLER_IMAGE_SHA256=$(echo "${MACHINE_OS_IMAGE_JSON}" | jq -r '.images.openstack.sha256')
```

Comment:
The caching code requires the uncompressed sha256 hash. e.g images.qemu.uncompressed-sha256.

Reply:
Thanks, I adjusted that in the latest version and added a FIXME referencing https://github.com/openshift/installer/issues/2845